docs/user/tutorial-websockets.rst

```diff
@@ -0,0 +1,608 @@
+.. _tutorial-websockets:
+
+Tutorial (WebSockets)
+=====================
+
+In this tutorial, we're going to build a WebSocket server using Falcon. We'll start with a simple server that echoes back any message it receives.
```

Comment:
It would be nice to wrap these RST lines to ~80 characters (although it is not as important as in the case of source code).

Reply:
Done in f8a08640cb9ea7f75bad116ff6e5c32bf9840fcb. Not all lines fit without affecting the readability (IMO), but most do.